pygmt/base_plotting.py

```diff
@@ -816,6 +816,175 @@ def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("plot", arg_str)
 
+    @fmt_docstring
+    @use_alias(
+        A="straight_line",
+        B="frame",
+        C="cmap",
+        D="offset",
+        G="color",
+        I="intensity",
+        J="projection",
+        Jz="zscale",
+        JZ="zsize",
+        L="close",
+        N="no_clip",
+        Q="no_sort",
```

Comment:
Note that error_bar(E) is available in `plot`, but not in `plot3d`. Also, no_sort ([Q](https://docs.generic-mapping-tools.org/latest/plot3d.html#q)) is a new alias specific to `plot3d` only (not found in GMT.jl either)!